crates/std_detect/src/detect/os/macos/aarch64.rs

```diff
@@ -0,0 +1,98 @@
+//! Run-time feature detection for aarch64 on macOS.
+
+use crate::detect::{cache, Feature};
+
+#[inline]
+fn _sysctlbyname(name: &str) -> bool {
+    use libc;
+
+    let mut enabled: i32 = 0;
+    let mut enabled_len: usize = 4;
```

Comment:
Are you sure this is the correct size? I couldn't find anything in Apple's documentation that indicates what size to use.

Reply:
Other proeject such as [Google cpufeature](https://github.com/google/cpu_features/blob/main/src/impl_aarch64_macos_or_iphone.c) and [Apple WebKit](https://searchfox.org/wubkat/source/Source/JavaScriptCore/assembler/CPU.cpp#127) uses `int`.